### ISSUE TITLE
Review and fix Grammarly extension conflict

### ISSUE THREAD

**Issue by michaelhays** [dylantientcheu/urplus#10]

**Comment by WittmannF** [comment]
Any hint on what's causing this conflict?

**Comment by michaelhays** [comment]
None, and unfortunately I don't have the means to figure it out without a reviewer account. If someone is willing to give me access to their account so I can check it out, I'd be glad to (but to be honest, this would probably be the last "major fix" I have the time to help with).

**Comment by WittmannF** [comment]
You can access my account. Is a TOKEN enough for debugging or do you need username and password?

**Comment by dylantientcheu** [comment]
I will check this out after work.
Issue is, it can only be checked when performing a review

**Comment by michaelhays** [comment]
Thanks Dylan, let me know if I can assist in any way

**Comment by dylantientcheu** [comment]
Hi, I need to clarify this, the conflict is; the text disappears when typing. Right?

**Comment by michaelhays** [comment]
AFAIK, yes. I never used it long so I'm not sure if there's anything else

**Comment by WittmannF** [comment]
That's correct @blurdylan! I heavily use Grammarly, however, when both extensions are active, the text disappears.

**Comment by WittmannF** [comment]
https://www.useloom.com/share/cb96c86c44a347a480a5d7077ec3db2a

**Comment by dylantientcheu** [comment]
This is very difficult for me to work on, because I do not have many reviews these days and the 12 hours delay makes it harder.
@michaelhays props to you on building this app this way 😅.
Idk if you have a review simulator, it'll be of great use.

**Comment by WittmannF** [comment]
Does sharing .HTML saved webpages help @blurdylan?
